ansible/roles/stack-sunbird/templates/sl-assessments.env

```diff
@@ -0,0 +1,97 @@
+
+# CSV Config
+# Create the respective directory as provided in the path below
+CSV_REPORTS_PATH={{sl_assessments_csv_reports_path | default("reports")}}
+PUBLIC_FOLDER_PATH={{sl_assessments_public_folder_path | default("public")}}
+
+# Kafka Configuration
+KAFKA_COMMUNICATIONS_ON_OFF={{sl_assessments_kafka_communications_on_off | default("ON")}}
+KAFKA_URL ={{sl_assessments_kafka_url}} 
+COMPLETED_SUBMISSION_TOPIC={{sl_assessments_completed_submission_topic | default("ntpprod.sl.submission.raw")}}
+INCOMPLETE_SUBMISSION_TOPIC={{sl_assessments_incomplete_submission_topic | default("ntpprod.sl.incomplete.submission.raw")}}
+SUBMISSION_RATING_QUEUE_TOPIC={{sl_assessments_submission_rating_queue_topic | default("ntpprod.sl.submission.rating.raw")}}
+COMPLETED_OBSERVATION_SUBMISSION_TOPIC={{sl_assessments_completed_observation_submission_topic | default("ntpprod.sl.observation.raw")}}
+INCOMPLETE_OBSERVATION_SUBMISSION_TOPIC={{sl_assessments_incomplete_observation_submission_topic | default("ntpprod.sl.incomplete.observation.raw")}}
+NOTIFICATIONS_TOPIC={{sl_assessments_notifications_topic | default("ntpprod.sl.notification.raw")}}
+COMPLETED_SURVEY_SUBMISSION_TOPIC={{sl_assessments_completed_survey_submission_topic | default("ntpprod.sl.survey.raw")}}
+INCOMPLETE_SURVEY_SUBMISSION_TOPIC={{sl_assessments_incomplete_survey_submission_topic | default("ntpprod.sl.incomplete.survey.raw")}}
+KAFKA_ERROR_MESSAGES_TO_SLACK={{sl_assessments_kafka_error_messages_to_slack | default("ON")}}
+KAFKA_GROUP_ID ={{sl_assessments_kafka_group_id | default("samiksha")}}
+
+
+# Elastic search configurations 
+ELASTICSEARCH_COMMUNICATIONS_ON_OFF={{sl_assessments_elasticsearch_communications_on_off | default("ON")}}
+ELASTICSEARCH_HOST_URL={{sl_assessments_elasticsearch_host_url}}
+ELASTIC_SEARCH_REQUEST_TIMEOUT={{sl_assessments_elastic_search_request_timeout | default("60000")}}
+ELASTIC_SEARCH_MAX_RETRIES={{sl_assessments_elastic_search_max_retries | default("5")}}
+ELASTIC_SEARCH_SNIFF_ON_START={{sl_assessments_elastic_search_sniff_on_start | default("true")}}
+ELASTICSEARCH_USER_EXTENSION_INDEX={{sl_assessments_elasticsearch_user_extension_index}}
+ELASTICSEARCH_USER_EXTENSION_INDEX_TYPE={{sl_assessments_elasticsearch_user_extension_index_type | default("common")}}
+ELASTICSEARCH_ENTITIES_INDEX={{sl_assessments_elasticsearch_entities_index | default("entities")}}
+ELASTICSEARCH_ENTITIES_INDEX_TYPE={{sl_assessments_elasticsearch_entities_index_type | default("common")}}
+
+# KENDRA SERVICE
+KENDRA_APPLICATION_ENDPOINT={{sl_assessments_kendra_application_endpoint}}
+KENDRA_BASE_URL={{Sl_assessments_kendra_Base_url}}
+
+# OFFLINE TOKEN VALIDATION
+VALIDATE_ACCESS_TOKEN_OFFLINE={{sl_assessments_validate_access_token_offline | default("ON")}}
+KEYCLOAK_PUBLIC_KEY_PATH={{sl_assessments_keycloak_public_key_path)}}
+
+
+# IMPROVEMENT PROJECT SERVICE
+
+IMPROVEMENT_PROJECT_HOST={{sl_assessments_improvement_project_host}}
```

Comment:
What is this variable @aks30?

Reply:
same goes for all the following variables

Reply:
@rjshrjndrn - This variable will hold the value of  **Improvement project service** base url

Reply:
do we have that service in sunbird? or is it something external ?

Reply:
it is inside sunbird

Reply:
Again, if it's running along side other apps in kube, you can directly give the service name like kendra service

Reply:
shall i define URL like this http://improvement-service:7301

Reply:
sounds good. I hope there is a kubernetes service called improvement-service

Reply:
in current production i have checked there is no improvement-service over there,  shall i  define over here

Reply:
changes done rajesh